### PR TITLE
Fix that [utf16] reallocate memory; Improve code-style of unicode.mpl

### PR DESCRIPTION
**Pull request by myCrack**

### CODE REVIEW

File: windows/unicode.mpl [hunk 1]
```diff
@@ -9,39 +9,40 @@
...
     source.data storageAddress
     MB_ERR_INVALID_CHARS
-    CP_UTF8 MultiByteToWideChar
+    CP_UTF8
+    MultiByteToWideChar
   ;
+  [targetNat16Count 0 = ~] "MultiByteToWideChar failed" assert
 
-  [destSize 0 = ~] "MultiByteToWideChar failed" assert
+  result: Nat16 Array [targetNat16Count 1 + swap.enlarge] keep;
```
Comment (rmrfslashhome) [inline windows/unicode.mpl:32]: I suggest adding a space after `swap`, so that it does not look like a method call.

**Review by rmrfslashhome — CHANGES_REQUESTED** [review]

File: windows/unicode.mpl [hunk 2]
```diff
@@ -9,39 +9,40 @@
...
+  targetNat16Count:
     0
     0nx
     source.size
     source.data storageAddress
     MB_ERR_INVALID_CHARS
-    CP_UTF8 MultiByteToWideChar
+    CP_UTF8
+    MultiByteToWideChar
   ;
```
Comment (rmrfslashhome) [inline windows/unicode.mpl:29]: I am unsure if a semicolon should be placed on a separate line. The only places in sl where this is the case are here and in Process.mpl.

File: windows/unicode.mpl [hunk 3]
```diff
@@ -9,39 +9,40 @@
...
+  writtenNat16Count:
+    targetNat16Count new
     result.data storageAddress
     source.size
     source.data storageAddress
     MB_ERR_INVALID_CHARS
-    CP_UTF8 MultiByteToWideChar
+    CP_UTF8
+    MultiByteToWideChar
   ;
```
Comment (rmrfslashhome) [inline windows/unicode.mpl:42]: Likewise.

**Review by rmrfslashhome — CHANGES_REQUESTED** [review]

File: windows/unicode.mpl (hunk 2 above)
Comment (myCrack) [reply windows/unicode.mpl:29]: https://github.com/Matway/mpl-sl/blob/9d5b602d112bd3cf1ea0d328ef7d1c212631cc12/Union.mpl#L38-L48

File: windows/unicode.mpl (hunk 3 above)
Comment (myCrack) [reply windows/unicode.mpl:42]: Ditto.

File: windows/unicode.mpl (hunk 1 above)
Comment (myCrack) [reply windows/unicode.mpl]: Done - https://github.com/Matway/mpl-sl/pull/74/commits/e16aa2149ee5c4a266bb9a9a6907d5216e28925a.

**Pull request merged**